cmd/translatesfx/README.md

```diff
@@ -11,43 +11,57 @@ in some cases to produce OpenTelemetry Collector configs that are fully
 functional and comparable to the original Smart Agent config. However, 
 this tool is designed to produce only a reasonably accurate approximation of 
 the final, production Otel config that would replace a Smart Agent config. It is 
-not designed to, and cannot, produce a drop-in replacement Otel config for a
-Smart Agent one in all cases.
+not designed to, and cannot, in all cases produce a drop-in replacement Otel
+config for a Smart Agent one.
 
-This tool aims to remove a lot of the drudgery from migrating to Otel 
-from Smart Agent, but any config produced by this tool should be carefully 
+Instead, this tool aims to remove a lot of the bookkeeping from migrating to Otel 
+from Smart Agent; any config produced by this tool should be carefully 
 evaluated and tested before being put into production.
 
+## Where to Get It
+
+`translatesfx` executables are available on the
+[releases page](https://github.com/signalfx/splunk-otel-collector/releases).
+and are contained in the rpm, msi, and deb packages (v0.36.1 and up).
+
 ## Usage
 
 The `translatesfx` command requires one argument, a Smart Agent 
-configuration file, and accepts a second argument, the working directory 
-used by any Smart Agent `#from` directives. The `translatesfx` command uses 
-this working directory to resolve any relative paths to files referenced by 
-any `#from` directives. This working directory argument may be omitted, in 
-which case `translatesfx` expands relative file paths using the current 
-working directory.
+configuration file, and accepts an optional second argument, the working directory 
+used by any Smart Agent `#from` file expansion directives. The `translatesfx` command
+uses this working directory to resolve any relative paths to files referenced by 
+any `#from` directives at runtime. If this working directory argument is omitted, `translatesfx`
+expands relative file paths using the current working directory.
 
 ```
-> translatesfx <sfx-file> [<file expansion working directory>]
+% translatesfx <sfx-file> [<file expansion working directory>]
+```
+
+When `translatesfx` runs, it sends the translated Open Telemetry collector configuration
```

Comment:
- "OpenTelemetry collector" to "OpenTelemetry Collector"
- "stdout" to "standard output"
-  the approved abbreviation for OpenTelemetry is "OTel"

Reply:
Fixed.